assets/js/plugins/KimaiDatatable.js

```diff
@@ -79,11 +82,26 @@ export default class KimaiDatatable extends KimaiPlugin {
                     jQuery(html).find(contentArea)
                 );
                 durations.updateRecords();
+                self.fixDropdowns();
             },
             error: function(xhr, err) {
                 form.submit();
             }
         });
 
     }
+
+    fixDropdowns() {
+        // HACK: show dropdown menu upwards, if it "leaves" the page
+        jQuery(this.selector + ' [data-toggle=dropdown]').each(function() {
+            let parent = jQuery(this).parent();
+            let menu = parent.find('.dropdown-menu');
+
+            if (parent && menu) {
```

Comment:
I am not a fan of these implicit checks... I guess this case will rarely happen, I'd just orefer something like `parent !== undefined`.

Reply:
i also want to implicitly check for empty objects/arrays, since this is most probably returned by jquery ...
If you want to make it explicit, then we have to check that too :thinking:

Reply:
I am a completely javascript noob and never used this kind of checks neither in JS nor PHP, so maybe I am just a bit over complicating here. If this surely works, fine for me as well.